src/libs/strings.ts

```diff
@@ -67,11 +67,13 @@ export const strings = {
     nowVerifying: "Now Verifying..",
     sending: "Sending...",
     sendConfirmation: "Send Confirmation",
-
+    dailyAchievements: "Daily Achievements",
```

Comment:
Yesss! Great catch to add the new text & directions via I18n instead of directly to the component file.